data/coalition/coalition missions.txt

```diff
@@ -1139,7 +1139,7 @@ mission "Saryd Couple 3"
 	destination "Saros"
 	on offer
 		conversation
-			`The couple leaves your ship and heads to a real estate agency. From what they told you during the trip, they have about 1,500,000 credits they can spend, which they claim is in line with the standard properties in the most valued (and most sought-after) locations here, and that they should still have enough money to pay you afterwards.`
+			`The couple leaves your ship and heads to a real estate agency. From what they told you during the trip, they have about two and a half million credits they can spend, which they claim is in line with the standard properties in the most valued (and most sought-after) locations here, and that they should still have enough money to pay you afterwards.`
```

Comment:
Searching the data for `million` and `,000` found only one other instance of the latter being used for a value of a million or more, and that was an exact payment amount. For the former, words (`two and a half million`) occur slightly more than digits (`2.5 million`) although none of the existing cases had fractional amounts (so `ten million` vs `10 million`).

Feel free to change the formatting to whatever version makes the most sense, though -- I care far more about the size of the budget than its textual representation. :-)